sdk/python/feast/repo_operations.py

```diff
@@ -114,6 +115,7 @@ def parse_repo(repo_root: Path) -> RepoContents:
         request_feature_views=[],
     )
 
+    res.entities.append(DUMMY_ENTITY)
```

Comment:
@Rostifar why is this needed?

Reply:
Yeah, this is an entity implemented for FeatureViews that don't have entities: checkout this https://github.com/Affirm/feast/blob/de3d7b8d36c8c5c08e372c3dbb078d17e8870274/sdk/python/feast/repo_operations.py#L201